code/modules/client/client procs.dm

```diff
@@ -537,6 +547,14 @@
 			message_admins("SQL ERROR during log_client_to_db (update). Error : \[[err]\]\n")
 	else
 		//New player!! Need to insert all the stuff
+
+		// Check new peeps for panic bunker
+		if(GLOB.panic_bunker_enabled)
+			message_admins("<span class='adminnotice'>Failed Login: [key] - New account attempting to connect during panic bunker</span>")
+			src << "Sorry but the server is currently not accepting connections from never before seen players."
```

Comment:
Adding a "Try again later" would be good, with the automatic stuff kyet said

Reply:
The automatic stuff is in, ill change this message